src/pages/VKeywordGenerator.vue

```diff
@@ -0,0 +1,112 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { MaButton, MaCard, MaInput, MaBadge } from '@mobileaction/action-kit';
+
+const inputText = ref('');
+const keywords = ref([]);
```

Comment:
```suggestion
const oneGramKeywords = ref([]);
const twoGramKeywords = ref([]);
const threeGramKeywords = ref([]);
```
My suggestion is use one object reference in `ref` as suggested in vue documentation. When you have complex structures in the ref value you could have reference/reactivity issues while you mutating them.